zerver/openapi/zulip.yaml

```diff
@@ -4744,7 +4748,15 @@ paths:
                             the user was subscribed to as a result of the query.
                           type: array
                           items:
-                            type: string
+                            type: object
+                            description: |
+                              A dictionary containing name and ID of the stream.
+                            additionalProperties: false
+                            properties:
+                              name:
+                                type: string
+                              id:
+                                type: integer
```

Comment:
Could this block somehow use `AddSubscriptionsResponse` rather than duplicating it?